livekit-plugins/livekit-plugins-assemblyai/livekit/plugins/assemblyai/stt.py

```diff
@@ -0,0 +1,408 @@
+# Copyright 2023 LiveKit, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import annotations
+
+import asyncio
+import dataclasses
+import json
+import os
+import time
+from contextlib import suppress
+from dataclasses import dataclass
+from typing import List, Optional, Union
+from urllib.parse import urlencode
+
+import aiohttp
+from livekit import rtc
+from livekit.agents import stt, utils
+from livekit.agents.utils import AudioBuffer
+
+
+ENGLISH = "en"
+
+
+@dataclass
+class STTOptions:
+    sample_rate: int | None = None
+    word_boost: str | None = None
+    encoding: str | None = None  # Allowed values: pcm_s16le, pcm_mulaw
+    disable_partial_transcripts: bool = False
+    enable_extra_session_information: bool = False
+    end_utterance_silence_threshold: int = None
+    buffer_size_seconds: int = None
+    token_expires_in: int = None
+
+
+class STT(stt.STT):
+    def __init__(
+        self,
+        *,
+        api_key: str | None = None,
+        sample_rate: int | None = 16000,
+        word_boost: str | None = None,
```

Comment:
I assume this is what the user sets, right? In that case I'd make it a `List[str] = []`. According to our docs, *"The parameter value must be a JSON encoded array of strings."*

In our SDK the user can set e.g. `word_boost = ["foo", "bar"]`, and under the hood we do:
`params["word_boost"] = json.dumps(self._word_boost)`, and later urlencode the params. Compare with Python SDK code: https://github.com/AssemblyAI/assemblyai-python-sdk/blob/master/assemblyai/transcriber.py#L1094.

A list will be easier for the user experience, otherwise they have to ensure this is already in the correct string format, e.g `'["foo", "bar"]'`